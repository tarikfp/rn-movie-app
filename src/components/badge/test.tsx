import React from "react";
import { TestUtils } from "~utils";
import Badge from "./component";

describe("Tests badge component", () => {
  it("renders badge correctly", () => {
    const tree = TestUtils.createWithWrappers(<Badge label="test" />, {
      hasTheme: true,
    });

    expect(tree).toMatchSnapshot();
  });
});
