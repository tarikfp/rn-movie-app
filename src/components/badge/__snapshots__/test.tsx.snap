// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests badge component renders badge correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#7C92FF",
        "borderRadius": 16,
        "height": 24,
        "justifyContent": "center",
        "marginRight": 6,
        "marginTop": 6,
        "minWidth": 105,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "backgroundColor": "#7C92FF",
          "color": "#FFFFFF",
          "fontFamily": "Poppins-Light",
          "fontSize": 18.857142857142858,
          "fontWeight": "400",
        },
      ]
    }
  >
    test
  </Text>
</View>
`;
